### ISSUE TITLE
Failed to store images inside replicator

### ISSUE THREAD

**Issue by maxwkfcc** [duncanmcclean/guest-entries#33]
### Description

I am using Guest Entries addon.  I see that I can following the guideline here to use replicator (https://guest-entries.duncanmcclean.com/examples/using-with-replicator).

My case is I am having a file upload element inside a replicator that requires multiple file uploads.  How should I do that?

I am using this line of code.
```
<input class="form-control" type="file" name="rich_text_and_image[0][gallery[]]" multiple="">
```

But after the form submission, I go to the collection and cannot find any images being uploaded. 

### Steps to reproduce

1. Create a collection blueprint with a replicator item
2. Inside the replicator item, add file upload element
3. Create a form using Guest Entry with file upload element to the replicator element
```
<!--
rich_text_and_image is the handle of the replicator 
gallery is the file upload element handle
-->
<input class="form-control" type="file" name="rich_text_and_image[0][gallery[]]" multiple="">
```
4. In the form, add image and submit
5. Go back to control panel and check if the files are being added
6. Cannot see any files

### Environment

Statamic 3.3.51 Pro
Laravel 8.83.23
PHP 8.1.13
cc/guest-entries-support dev-main
doublethreedigital/guest-entries 1.2.2
jonassiewertsen/statamic-live-search 1.5.1
jonassiewertsen/statamic-livewire 2.9.0
mitydigital/sitemapamic 2.3.3
statamic/eloquent-driver 0.3.0
swiftmade/statamic-clear-assets 1.1.0
takepart-media/statamic-oreos 1.2.0

**Comment by maxwkfcc** [comment]
Hi Duncan,

Thank you for your update.  I have updated and tested on it.

While I am using `rich_text_and_image[0][gallery]`, the addon does capture the picture I uploaded and found in the entry.  However, it can only capture 1 file even I have selected multiple.

Then, I tried to rename the field to `rich_text_and_image[0][gallery[]]` and it is not working at all.

See if you have any ideas on it.

Thanks
Max
